api/proto/ank_base.proto

```diff
@@ -211,6 +211,13 @@ message WorkloadState {
     ExecutionState executionState = 2; /// The workload execution state.
 }
 
+/**
+* In case of workload names, the naming convention states that thier names shall:
+*  - contain only regular upper and lowercase characters (a-z and A-Z), numbers and the symbols "-" and "_"
+*  - have a minimal length of 1 character
+*  - have a maximal length of 63 characters
+* Also, agent name shall contain only regular upper and lowercase characters (a-z and A-Z), numbers and the symbols "-" and "_".
+*/
 message WorkloadInstanceName {
```

Comment:
This object is only sent by Ankaios and never received from a user of the Control Interface. 

The proper place to describe the limitation is a bit hidden and would be the `WorkloadMap` message.

Reply:
Done.

Reply:
Done.